src/main/java/com/theocean/fundering/domain/celebrity/service/CelebService.java

```diff
@@ -0,0 +1,42 @@
+package com.theocean.fundering.domain.celebrity.service;
+
+import com.theocean.fundering.domain.celebrity.domain.Celebrity;
+import com.theocean.fundering.domain.celebrity.dto.*;
+import com.theocean.fundering.domain.celebrity.repository.CelebRepository;
+import com.theocean.fundering.global.errors.exception.Exception400;
+import com.theocean.fundering.global.errors.exception.Exception500;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class CelebService {
+    private final CelebRepository celebRepository;
+    @Transactional
+    public void register(CelebRequestDTO celebRequestDTO) {
+        try{
+            celebRepository.save(celebRequestDTO.getEntity());
+        }catch (Exception e){
```

Comment:
```suggestion
        }catch (RuntimeException e){
```
RuntimeException 만으로 충분합니다.